crates/rust-analyzer/src/main_loop.rs

```diff
@@ -1038,4 +1041,43 @@ impl GlobalState {
             .finish();
         Ok(())
     }
+
+    fn fetch_aux_files(&mut self) {
```

Comment:
This is still not the right way to handle this. We need to make sure these files are registered to the file watcher, which will then handle the rest for us instead of us loading the files once at startup.